config-linux.md

```diff
@@ -641,7 +641,9 @@ The values MUST be absolute paths in the [container namespace](glossary.md#conta
 
 ## <a name="configLinuxMountLabel" />Mount Label
 
-**`mountLabel`** (string, OPTIONAL) will set the Selinux context for the mounts in the container.
+**`mountLabel`** (string, OPTIONAL) will set the SELinux context for the mounts in the container.
+If `mountLabel` is specified and SELinux is enabled, the runtime MUST set `mountLabel` for the mounts in the container.
```

Comment:
I don't think we want “and SELinux is enabled”.  Runtimes that don't support SELinux [will not be compliant][1], just like runtimes that don't support cgroups will not be compliant.  These runtimes might still be *useful* in certain contexts, and not all hosts may support SELinux (in which case configs that set `mountLabel` will likely crash in `create`), but in order to be a compliant runtime, you *MUST* support this property.  If we want compliance labeling for runtimes that don't support this property, we'd need a new [`selinux` compliance track][2].

And instead of the still-unclear-to-me “MUST set `mountLabel` for the mounts”, I'd rather focus on how we test that this happened.  For example:

    If `mountLabel` is set, `lgetxattr(2)` with `security.selinux` for `name` MUST return this value when called from the container mount namespace (or context, or whatever) on `/` and all `mounts[].destination`.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/spec.md#notational-conventions
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/spec.md#platforms